WMF Framework/WMFOnThisDayEventsFetcher.m

```diff
@@ -0,0 +1,65 @@
+#import "WMFOnThisDayEventsFetcher.h"
+#import "WMFFeedOnThisDayEvent.h"
+#import <WMF/WMF-Swift.h>
```

Comment:
@joewalsh 
Hmm... for some reason it doesn't compile for me using that syntax in that file...